src/integrationTest/java/uk/gov/hmcts/darts/audio/controller/AudioRequestsControllerDownloadIntTest.java

```diff
@@ -109,16 +104,6 @@ void audioRequestDownloadShouldDownloadFromOutboundStorageAndReturnSuccess() thr
             transformedMediaId,
             Set.of(TRANSCRIBER)
         );
-
-        AuditSearchQuery searchQuery = new AuditSearchQuery();
-        searchQuery.setCaseId(mediaRequestEntity.getHearing().getCourtCase().getId());
-        searchQuery.setFromDate(OffsetDateTime.now().minusDays(1));
-        searchQuery.setToDate(OffsetDateTime.now().plusDays(1));
-        searchQuery.setAuditActivityId(DOWNLOAD_AUDIT_ACTIVITY_ID);
-
-        List<AuditEntity> auditEntities = auditService.search(searchQuery);
-        assertEquals("2", auditEntities.get(0).getCourtCase().getCaseNumber());
-        assertEquals(1, auditEntities.size());
```

Comment:
This needs to be tested in a different way, not just removed.

Reply:
Code updated for tests